src/main/java/com/schematica/client/renderer/RenderSchematic.java

```diff
@@ -572,13 +636,13 @@ public int renderBlockLayer(@NotNull final EnumWorldBlockLayer layer, final doub
             final RenderChunk renderChunk = renderInfo.renderChunk;
             final RenderOverlay renderOverlay = renderInfo.renderOverlay;
 
-            if (!renderChunk.getCompiledChunk().isLayerEmpty(layer))
+            //if (renderChunk.getCompiledChunk().isLayerEmpty(layer))
             {
                 count++;
                 this.renderContainer.addRenderChunk(renderChunk, layer);
             }
 
-            if (isTranslucent && renderOverlay != null && !renderOverlay.getCompiledChunk().isLayerEmpty(layer))
+            //if (isTranslucent && renderOverlay != null && !renderOverlay.getCompiledChunk().isLayerEmpty(layer))
             {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested code block into a method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1199)